### DISCUSSION TITLE
Installing from custom repository

### DISCUSSION THREAD

**Discussion by skoenfaelt** [yarnpkg/berry#5412] [Q&A] [answered]
Hello,
this use case is new to me, I want to install the JS package from [HereMaps](https://developer.here.com/documentation/maps/3.1.41.0/dev_guide/topics/angular-practices.html)

I thought the equvivalent of the
```
 npm config set @here:registry https://repo.platform.here.com/artifactory/api/npm/maps-api-for-javascript/
 npm install @here/maps-api-for-javascript
```

is according to the yarn 2 docs https://yarnpkg.com/cli/add
eg: `yarn add lodash@https://github.com/lodash/lodash`

I tried this:

```
yarn add here@https://repo.platform.here.com/artifactory/api/npm/maps-api-for-javascript
```

Unfortunately, this does not work and does not appear to be correct. What am I doing wrong?

The Error I got

`Error: here@https://repo.platform.here.com/artifactory/api/npm/maps-api-for-javascript/ isn't supported by any available resolver`

**Comment by merceyz** [comment, marked as answer]
The equivalent of those commands would be:
```sh
yarn config set "npmScopes.here.npmRegistryServer" "https://repo.platform.here.com/artifactory/api/npm/maps-api-for-javascript"
yarn add @here/maps-api-for-javascript
```

what you tried is for git repositories.

**Reply by skoenfaelt** [reply]
https://yarnpkg.com/cli/config/set
ahhh this documentation is according to that. Thank you very much for the quick and very helpful response!
